src/pages/my/todo.tsx

```diff
@@ -1,30 +1,119 @@
 import { GetServerSideProps } from 'next';
-import { Heading } from '@adobe/react-spectrum';
 import { FormattedMessage as Msg } from 'react-intl';
+import { useQuery } from 'react-query';
+import { Heading, Text, View } from '@adobe/react-spectrum';
 
+import CallAssignmentList from '../../components/CallAssignmentList';
+import EventList from '../../components/EventList';
+import getBookedEvents from '../../fetching/getBookedEvents';
+import getCallAssignments from '../../fetching/getCallAssignments';
+import getRespondEvents from '../../fetching/getRespondEvents';
 import MyHomeLayout from '../../components/layout/MyHomeLayout';
 import { PageWithLayout } from '../../types';
 import { scaffold } from '../../utils/next';
+import { useRespondEvents } from '../../hooks';
 
 const scaffoldOptions = {
     localeScope: [
         'layout.my',
+        'misc.callAssignmentList',
+        'misc.eventList',
         'misc.publicHeader',
         'pages.myTodo',
     ],
 };
 
-export const getServerSideProps : GetServerSideProps = scaffold(async () => {
-    return {
-        props: {},
-    };
+export const getServerSideProps : GetServerSideProps = scaffold(async (context) => {
+    const { user } = context;
+
+    let respondEventsState;
+    let bookedEventsState;
+    let callAssignmentsState;
+
+    if (user) {
```

Comment:
What happens if a non-authenticated user (i.e. when `user` is `null`) tries to access "/my" pages? It shouldn't be possible. Set the `authLevelRequired` scaffold option to tap into functionality that will automatically redirect anonymous users to login.